plato/draw/pythreejs/Scene.py

```diff
@@ -144,16 +149,23 @@ def enable(self, name, auto_value=None, **parameters):
             self.disable(name, strict=False)
             self._backend_objects['scene'].add(light)
         elif name == 'directional_light':
+
             (width, height) = self.size
             dz = np.sqrt(np.sum(self.size**2))*self._clip_scale
 
             lights = parameters.get('value', DEFAULT_DIRECTIONAL_LIGHTS)
             self.disable(name, strict=False)
+            if not self._use_default_lights:
+                for light in self._default_lights:
+                    self._backend_objects['camera'].remove(light)
```

Comment:
@bdice Making sure that the lights properly changed when enabling the feature multiple times wouldn't have been something I checked when first implementing this backend; could you put this removal logic inside `disable` for `directional_light`? For consistency, I guess the list should be stored as `self.backend_objects['lights']` or so, instead of a separate list inside the object (`_default_lights`). Then we don't even have to keep track of a `_use_default_lights` flag and can just enable the lights by default.

Reply:
I think I have addressed this adequately with commit fb16edbc4d05a2b7552f048cb6a6a680e93c492b.